docs/running_benchmarks.rst

```diff
@@ -37,7 +37,7 @@ For example,
 
 .. code-block::
 
-    nwb_benchmarks run --bench time_remote_slicing
+    nwb_benchmarks run --bench time_remote_file_reading
```

Comment:
I also swapped this demo instruction since the file reading benchmarks are faster and more verbose (providing more visual feedback)